scripts/staging-reset.js

```diff
@@ -0,0 +1,117 @@
+'use strict'
+
+const fs = require('fs')
+const { promisify } = require('util')
+const execute = promisify(require('child_process').exec)
+const replace = require('replace-in-file')
+
+const CI_FILE = '.gitlab-ci.yml'
+const REPOSITORY = 'git@github.com:DataDog/browser-sdk.git'
+const MAIN_BRANCH = 'main'
+
+const CURRENT_STAGING_BRANCH = process.env.CURRENT_STAGING
```

Comment:
I don't get where `CURRENT_STAGING` is defined 🤔

Reply:
At the top of gitlab-ci.yml as a variable

Reply:
oh it was added previously for to-staging support 🤦